src/main/scala/org/broadinstitute/dsde/workbench/sam/service/PolicyEvaluatorService.scala

```diff
@@ -48,6 +58,36 @@ class PolicyEvaluatorService(
     }
   }
 
+  /**
+    * Check if the user has the action on the resource.  A true result means they do, and can be trusted.  A false result only means that the short
+    * circuit approach did not find that to be true, but a more exhaustive check might.
```

Comment:
replace short circuit in comment with better name

Reply:
replaced with 'shallow check' here, and another reference to 'short circuit' as well